### PR TITLE
 work on Updating Profile

### PR DESCRIPTION
**Pull request by SIBONIYO**
#### What does this PR do?
  - Updating Profile
#### Description of Task to be completed?
 - The Operator and Driver should see their profile
 - The operator and driver should be able to update information about their own profile.
 - Appropriate error messages are displayed on unsuccessful attempts. 
 - Skeleton UIs/screen are/is shown when the page loads/is loading.
 - The operator and driver profile should be able accessible publicly.
 - The page is responsive.
#### How should this be manually tested?
- checkout this branch
- run npm install
- sign in as user and then go to the profile
- click Edit profile
#### What are the relevant trello id?
pgipeAIZ

### CODE REVIEW

File: src/tests/redux/actionCreators/updateProfileAction.test.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+          success: true,
+          message: 'profile Vewed successfully',
+          data: {}
+        }
+      });
+    });
+    await store.dispatch(viewProfileAction());
+    await flushPromises();
+    const calledActions = store.getActions();
+    // expect(calledActions[0].type).toEqual(VIEW_PROFILE);
```
Comment (raymond42) [inline src/tests/redux/actionCreators/updateProfileAction.test.js]: remove this commented line

File: src/tests/redux/actionCreators/updateProfileAction.test.js [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+          success: true,
+          message: 'profile Viewed successfully',
+          data: {}
+        }
+      });
+    });
+    await store.dispatch(viewProfileAction());
+    await flushPromises();
+    const calledActions = store.getActions();
+    // expect(calledActions[2].type).toEqual(VIEW_SUCCESS);
```
Comment (raymond42) [inline src/tests/redux/actionCreators/updateProfileAction.test.js]: remove the commented line

File: src/tests/redux/actionCreators/updateProfileAction.test.js [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+          success: true,
+          message: 'profile updated successfully',
+          data: {}
+        }
+      });
+    });
+    await store.dispatch(updateProfileAction());
+    await flushPromises();
+    const calledActions = store.getActions();
+    // expect(calledActions[2].type).toEqual(UPDATE_SUCCESS);
```
Comment (raymond42) [inline src/tests/redux/actionCreators/updateProfileAction.test.js]: remove this commented line

**Review by raymond42 — CHANGES_REQUESTED** [review]
Good work @SIBONIYO only a few comments to work on.
1. Remove some commented lines
2. Let's have the card that displays the profile in the middle of the page

**Review by 41hulk — DISMISSED** [review]
LGTM

**Review by samuel776 — DISMISSED** [review]

**Review by KundwaBruno — DISMISSED** [review]

**Review by raymond42 — APPROVED** [review]

**Pull request merged**